### DISCUSSION TITLE
ReactRouter v7 release date

### DISCUSSION THREAD

**Discussion by ondrejsevcik** [remix-run/react-router#12337] [Q&A] [answered]
Hello, we're starting a new project soon, and we need SSR. I'm deciding whether to go with Remix or wait for React Router v7.

Is there a target date for v7? I see there is already a 6th pre-release version, which makes me hopeful that it will be released soon.

Thank you.

**Comment by RomanBaiocco** [comment, marked as answer]
You are in luck https://github.com/remix-run/react-router/releases/tag/react-router%407.0.0

**Reply by ondrejsevcik** [reply]
That's great 😄 Christmas came early!
